python-searcher/python_searcher/searcher-cli.py

```diff
@@ -0,0 +1,187 @@
+import time
+from typing import List
+
+from solders.keypair import Keypair
+from solders.rpc.responses import GetBalanceResp
+
+from searcher import get_searcher_client
+from generated.searcher_pb2 import (
+    ConnectedLeadersRequest,
+    NextScheduledLeaderRequest,
+    PendingTxSubscriptionRequest,
+    NextScheduledLeaderResponse,
+)
+from generated.searcher_pb2_grpc import SearcherServiceStub
+import click
+
+from solders.transaction import VersionedTransaction
+from solana.rpc.api import Client
+
+
+@click.group("cli")
+@click.pass_context
+@click.option(
```

Comment:
can use Paths for better error handling: https://click.palletsprojects.com/en/8.1.x/arguments/#file-path-arguments

Reply:
There's also an explicit File type if that works better: https://click.palletsprojects.com/en/8.1.x/api/#click.File